src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Config/EventHubExtensionConfigProvider.cs

```diff
@@ -72,6 +73,9 @@ public void Initialize(ExtensionConfigContext context)
             context.AddBindingRule<EventHubAttribute>()
                 .BindToCollector(BuildFromAttribute);
 
+            // register EventClient output binding
+            context.AddBindingRule<EventHubAttribute>().BindToValueProvider(GetEventClient);
```

Comment:
I think you should be using BindToInput instead. See other examples in other extensions, which use this for returning "clients". Here's an example: https://github.com/Azure/azure-webjobs-sdk-extensions/blob/b7aa8f84c2b3c9065861fe955ce8180b9d19f939/src/WebJobs.Extensions.CosmosDB/Config/CosmosDBExtensionConfigProvider.cs#L58

Reply:
So you'd have something like:

```charp
context.AddBindingRule<EventHubAttribute>()
    .BindToInput<EventHubClient>(a =>
    {
        return _options.Value.GetEventHubClient(a.EventHubName, a.Connection);
    });
```